Gemfile

```diff
@@ -19,6 +19,7 @@ gem "image_processing", "~> 1"
 gem "interactor", "~> 3"
 gem "kaminari", "~> 1"
 gem "notifications-ruby-client", "~> 3.1"
+gem "pdf-reader", "~> 2.2"
```

Comment:
It looks like it'd be more consistent with our other version pinning for this to be `~> 2`